tests/src/kinAccount.intg.test.ts

```diff
@@ -0,0 +1,156 @@
+import {KinAccount} from "../../scripts/src/kinAccount";
+import {Environment} from "../../scripts/src/environment";
+import {Channels, KeyPair, KinClient} from "../../scripts/src";
+import {Memo, Operation, Keypair, Network, Transaction as XdrTransaction} from "@kinecosystem/kin-base";
+import {Server} from "@kinecosystem/kin-sdk";
+
+const keyPair = KeyPair.generate();
+const seconedKeypair = KeyPair.generate();
+const thirdKeypair = KeyPair.generate();
+let client: KinClient;
+let account: KinAccount;
+let account2: KinAccount;
+
+describe("KinAccount", async () => {
+	beforeAll(async () => {
+		client = new KinClient(Environment.Testnet);
+		account = client.createKinAccount({seed: keyPair.seed});
```

Comment:
I think sender/receiver account might be less confusing what each account is responsible.